### DISCUSSION TITLE
Create Data

### DISCUSSION THREAD

**Discussion by mkiran18** [postmanlabs/postman-flows#335] [Q&A]
Hi Team,

Was looking for a block where i can aggregate output response from "Send Request" to a variable/Durable. Found this "Create Data" where it can accept two request data and out one variable.

![image](https://user-images.githubusercontent.com/30534081/174653664-02407875-94ef-42b9-a818-4a7c1ce87d5e.png)

Am not able add the second variable in the block, i tried restarting and nothing worked and am using the mac client

Any insights will be helpful

Thanks

**Comment by samay-v** [comment]
Hey @mkiran18 
Could you take a Snapshot of your Flow and upload it here? That'll give us more context on this issue
![Screenshot 2022-06-21 at 11 40 18 AM](https://user-images.githubusercontent.com/28357106/174728701-1018e621-2b42-425e-ad8c-eb1b42774a94.png)

**Reply by mkiran18** [reply]
Our flow contain PII information and as part of postman enterprise/cloud , here is the flow id : 62b094107bd0036b124e4770 or any other way to share the snapshot , please let me know

**Reply by mkiran18** [reply]
@samay-v any update on this ?
